docs/user/debugging.md

```diff
@@ -4,154 +4,243 @@
+* `/mnt/turbiniavolume/output/63de1268000f97d85198949e3fdb8dk5/1713143969-a34a3d37c4365d383c3f617169d04327-PlasoParserTask/worker-log.txt`
+```
 
+The above output gives us several paths that are useful for debugging:
+* The worker log (`worker-log.txt`) will show a subset of the logs from the Task
+  code.  This will have anything that is logged from the Task using
+  `TurbiniaResult.log`.
+* The stderr and stdout for all executions of binary dependencies (e.g. Plaso)
+  are available in the filenames prefixed with `stderr-` or `stdout-`
+  respectively.
+* Paths in `/tmp/` are the temporary locations for
+  output and are local to the Worker that executed that Task.  The hostname
+  for the Worker this Task was executed on is also listed in the output above.
 
-### Writing debug logs for binary dependencies
-Some Turbinia tasks execute binaries that can also write their own
-debug logs as part of execution.  This can be expensive so it is
-turned off by default.  You can specify `-T` during the request
-to enable these logs temporarily per-request  (e.g.
-`turbiniactl -T googleclouddisk -d disk-to-process`). Alternately
-you can set `DEBUG_TASKS = True` in the config file for the Worker.
-
-### Disappearing Tasks
-
-If the Turbinia server is scheduling Tasks, but you're not seeing the
-Tasks or their output for some reason, the most common scenario is that
-either a second Turbinia server or a second set of Workers is running
-and operating on the same PubSub queues.  Make sure that you have unique
-values per each Turbinia instance for the following config variables:
-`PSQ_TOPIC`, `INSTANCE_ID`, `PUBSUB_TOPIC` and `KOMBU_CHANNEL` (if you
-are using a local Turbinia installation type.
-
-## Common Errors
-
-*   If you encounter a cycle of errors on the Turbinia server when decoding
-    specific malformed pubsub messages, and the failure happens prior to the
-    server acknowledging the pubsub message, you may need to manually clear the
-    pubsub queue. If you are sure there aren't other outstanding requests, you
-    can just pull and auto-ack the existing messages, otherwise you may need a
-    variation of this command to selectively ack messages:
 
+To get all of the output files that have been saved to the `OUTPUT_DIR` you can run the following command:
 ```
-gcloud beta pubsub subscriptions pull --auto-ack $my-pubsub-subscription-name
+$ turbinia-client result task 5159b8915b5b4443bcbf3b6c2b9e7cf2
+Saving output for task 5159b8915b5b4443bcbf3b6c2b9e7cf2 to: 5159b8915b5b4443bcbf3b6c2b9e7cf2.tgz
 ```
+
+To get the files in /tmp on the Worker you'll need to get those from the Worker
+directly and the method to do so depends on the your instance configuration.  As
+an example, here is how you might gather the `worker-log.txt` from a worker
+running in a GKE/Kubernetes cluster.  Note that in this case we wouldn't
```

Comment:
Maybe move the Note part underneath the command for readability.

e.g.
> **Note**: The file can also be retrieved with `turbinia-client result task <task id>`, but as a last resort the `/tmp` directory can be checked for temporary files that did not get copied into the `OUTPUT_DIR`.

Reply:
I moved the note below the command and updated this section a bit, LMK what you think.